manifests/key_pair.pp

```diff
@@ -0,0 +1,31 @@
+define certs::key_pair (
```

Comment:
I think the very similar name is confusing but I don't know what a better name would be.

Can we also declare this as a private API?
```suggestion
# @summary Deploy a certificate and its matching key
# @api private
define certs::key_pair (
```

Reply:
- cert_key
- key_cert

If I migrate everything at once, I could drop the old `key_pair` and then it wouldn't clash. This is a bit of whether we want to try to do a piece at a time, or switch everything to this new method together.

Reply:
I think it makes sense to submit this as a small PR now, but we should really switch all at once IMHO. I think that's easier to deal with.

Reply:
Now that the PR removes the old one, can we rename it back to `cert::keypair`?